assets/chezmoi.io/docs/reference/configuration-file/variables.md.yaml

```diff
@@ -294,7 +294,7 @@ sections:
       description: KeePassXC database
     mode:
       default: '`cache-password`'
-      description: See section on KeePassXC
```

Comment:
For a reason I don't understand, this link doesn't work for me in local testing. But this PR is really good :)

Reply:
Should be fixed now.